Makefile

```diff
@@ -62,16 +67,14 @@ lint-with-docker:  ## Lint source code with docker image of golangci-lint
 .PHONY: lint lint-with-local lint-with-docker
 
 # --- Docker -------------------------------------------------------------------
-DOCKER_TAG ?= $(error DOCKER_TAG not set)
 
 docker-build:
 	docker build --tag foxtrot:latest .
 
 docker-build-release:
 	docker buildx build \
-		--push \
+		$(if $(SEMVER),--push --tag foxygoat/foxtrot:$(SEMVER)) \
```

Comment:
When I wrote that I assumed that SEMVER might be empty. But we set it unconditionally. I think perhaps it might be best to keep DOCKER_TAG then and set it from the CI workflow so it will be empty if we run this locally. Otherwise we risk pushing dev builds. What do you think?

Reply:
Agreed.